cmake/umd_device.cmake

```diff
@@ -14,11 +14,6 @@ set(UMD_SRC
     ${UMD_HOME}/device/wormhole_implementation.cpp
 )
 
-add_library(umd_device OBJECT ${UMD_SRC})
+add_library(umd_device STATIC ${UMD_SRC})
```

Comment:
Why static?

Reply:
Well, because that's the only way we use it. You only need an object library in case you need to specify the same object files multiple times, like when you want to compile both a static and a shared version of a library. Otherwise, there's no reason to use it. Here we just compile all the files into a single static lib and link it as necessary.